tests/data/tensorflow_mnist/mnist.py

```diff
@@ -10,114 +10,14 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
-from __future__ import absolute_import, division, print_function
-
```

Comment:
This script has been moved to `mnist_v1.py`, and has been replaced with a script that calls the appropriate mnist training script based on the framework version on the container being used to run the SM Training Job.